package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "@google-cloud/common": "^0.12.0",
+    "cloud-trace-counter": "^3.0.0",
```

Comment:
would be better if the dependency name could be a function of the tracked package name. I.e. possible to write a function that would convert `@google-cloud/trace-agent` into the name of the package counter module. Something like counter_google-cloud_trace-agent?